pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/ConnectionPoolBrokerRequestHandler.java

```diff
@@ -232,6 +237,19 @@ protected BrokerResponse processBrokerRequest(long requestId, BrokerRequest orig
     return brokerResponse;
   }
 
+  // For backward-compatible
+  private Map<String, List<String>> convertToOldRoutingTable(
```

Comment:
why do we need this?

Reply:
The connection-pool routing is already deprecated, and is subject to be removed, thus here we just keep it backward-compatible.